src/WebJobs.Extensions.DurableTask/LinuxLogger.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.Extensions.Logging;
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask
+{
+    public class LinuxLogger : ILogger
+    {
+        private readonly bool inConsumption;
+        private readonly string consolePrefix = "MS_DURABLE_FUNCTION_EVENTS_LOGS";
+        private readonly string loggingPath = "/var/log/functionsLogs/durableevents.log";
+
+        public LinuxLogger(bool inConsumption)
+        {
+            this.inConsumption = inConsumption;
+        }
+
+        public IDisposable BeginScope<TState>(TState state)
+        {
+            // TODO: I this we can leave this unimplemented
+            throw new NotImplementedException();
```

Comment:
It might not be a good idea to throw here because some code we don't control could theoretically invoke this and cause issues. We might want to do some exploratory testing and see whether it functions correctly if you just return `null`.

Reply:
Hmm, I'm not the biggest fan of adding `null`s but I trust your judgement here. Let's leave this issue as unresolved so we can do some testing for it

Reply:
`null` is just the lazy way to go (and I tend to be lazy sometimes :D). If you want to make this 100% robust, you can instead implement a no-op `IDisposable` class and return a static instance of that.

Reply:
Something like [this](https://github.com/StephenCleary/Disposables/blob/master/src/Nito.Disposables/NoopDisposable.cs) would work.